OUDS/Core/Tokens/RawTokens/Sources/BorderRawTokens.swift

```diff
@@ -15,17 +15,17 @@ import Foundation
 
 // MARK: - Type aliases to keep grammar clear
 
-/// Typeliases precising `Double` value are used (because of multiplier factors) for a **border width raw token**.
+/// Type alias precising `Double` value are used (because of multiplier factors) for a **border width raw token**.
 public typealias BorderWidthRawToken = Double
 
-/// Typeliases precising `Double` value are used (because of multiplier factors) for a **border widthradiusraw token**.
+/// Type alias precising `Double` value are used (because of multiplier factors) for a **border widthradiusraw token**.
```

Comment:
I haven't modified "widthradiusraw". You can update it by adding a new commit in this PR if needed :)